### PR TITLE
Add hover state and inbetween breakpoint to new nav

### PR DESCRIPTION
**Pull request by davidrapson**
This PR adds hover styles for the drop-down nav in the new header and adds "inbetween" breakpoint styles as well as generally refining things.

hover

screen shot 2018-08-03 at 11 06 04

This brings us to the point where the navigation prototype is largely "done" in that it's inline with design. The css and templates behind this could definitely stand to be cleaned up but that can wait until we've run some tests with this.

### CODE REVIEW

File: views/includes/metaHead.njk [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <link rel="stylesheet" type="text/css" href="{{ 'stylesheets/style.css' | getCachebustedPath }}">
 <link href="https://fonts.googleapis.com/css?family=Poppins:300,500,600|Roboto:400,700" rel="stylesheet">
 
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=HTMLPictureElement,Promise,Array.prototype.find"></script>
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=HTMLPictureElement,Promise,Array.prototype.find,Element.prototype.closest"></script>
```
Comment (davidrapson) [inline views/includes/metaHead.njk]: I've added `Element.prototype.closest` but all but IE11 supports it, and it's unlikely IE11 users would trigger this so we might be able to get away without this.

File: views/includes/global-header-next.njk [hunk 2]
```diff
@@ -150,3 +150,60 @@
         </div>
     </div>
 </header>
+
+<script>
```
Comment (davidrapson) [inline views/includes/global-header-next.njk]: Slightly controversial but this is the quickest way to add event listeners for these elements, by triggering the script here the part of the DOM we care about is ready and it means users can click the navigation links before the rest of the page has finished resulting in a snappy time to interactive for common navigation tasks. 💭 thought's welcome on if there is a nicer way to approach this.

File: views/includes/global-header-next.njk [hunk 3]
```diff
@@ -54,6 +66,28 @@
     </ul>
 {% endmacro %}
 
+{% macro globalSearch() %}
+    <form class="global-search-next" role="search" action="/search">
+        <input type="hidden" name="lang" value="en-GB">
```
Comment (mattandrews) [inline views/includes/global-header-next.njk]: Just looking at this... wonder if we still need these parameters now the search goes to Google?

File: views/includes/global-header-next.njk [hunk 4]
```diff
@@ -103,8 +145,65 @@
...
+        nav: 'has-toggled-navigation',
+        search: 'has-toggled-search'
+    };
+
+    var body = document.querySelector('body');
+    var html = document.querySelector('html');
+    var el = document.querySelector(elSelector);
+    var searchInput = el.querySelector('input[type=search]');
+
+    if (!el) {
```
Comment (mattandrews) [inline views/includes/global-header-next.njk]: You could maybe move this up a bit earlier and bail out before doing the other queries?

**Pull request merged**